src/plugins/intel_cpu/src/transformations/snippets/common/pass/mul_add_to_fma.cpp

```diff
@@ -15,7 +15,6 @@
 #include "transformations/utils/utils.hpp"
 
 ov::intel_cpu::pass::MulAddToFMA::MulAddToFMA() {
-    MATCHER_SCOPE(MulAddToFMA);
```

Comment:
Please double check if it is really necessary

Reply:
Checked, reused previously unused variable generated under the hood of this macro